tests/e2e/test_diffusion.py

```diff
@@ -1106,6 +1138,68 @@ def test_diffusion_lora(
     ensure_similar_images(predicted_image, expected_image, min_psnr=35, min_ssim=0.98)
 
 
+@no_grad()
+def test_diffusion_sdxl_batch2(sdxl_ddim: StableDiffusion_XL) -> None:
```

Comment:
Any reason why you did not write `test_diffusion_batch2` (for SD1) the same way? i.e. just one diffusion step + comparison with/without batching?

Reply:
no good reasons, fixed now